ESPixelStick/src/input/InputFPPRemotePlayFile.cpp

```diff
@@ -283,27 +289,49 @@ bool c_InputFPPRemotePlayFile::ParseFseqFile ()
                 CurrentSparseRange.DataOffset   = read24 (FseqRawRanges[SparseRangeIndex].Start);
                 CurrentSparseRange.ChannelCount = read24 (FseqRawRanges[SparseRangeIndex].Length);
                 TotalChannels += CurrentSparseRange.ChannelCount;
-                LargestBlock  = max (LargestBlock, CurrentSparseRange.ChannelCount);
+                LargestOffset = max (LargestOffset, CurrentSparseRange.DataOffset);
+                LargestBlock  = max (LargestBlock, CurrentSparseRange.DataOffset + CurrentSparseRange.ChannelCount);
 
-                // DEBUG_V (String ("            RangeChannelCount: ") + String (CurrentSparseRange.ChannelCount));
-                // DEBUG_V (String ("              RangeDataOffset: 0x") + String (CurrentSparseRange.DataOffset, HEX));
+#ifdef DUMP_FSEQ_HEADER
+                DEBUG_V (String ("            RangeChannelCount: ") + String (CurrentSparseRange.ChannelCount));
+                DEBUG_V (String ("              RangeDataOffset: 0x") + String (CurrentSparseRange.DataOffset, HEX));
+#endif // def DUMP_FSEQ_HEADER
 
                 ++SparseRangeIndex;
             }
 
-            // DEBUG_V (String ("                TotalChannels: ") + String (TotalChannels));
-            // DEBUG_V (String ("                 LargestBlock: ") + String (LargestBlock));
-            if (TotalChannels > fsqParsedHeader.channelCount)
+#ifdef DUMP_FSEQ_HEADER
+            DEBUG_V (String ("                TotalChannels: ") + String (TotalChannels));
+            DEBUG_V (String ("                LargestOffset: ") + String (LargestOffset));
+            DEBUG_V (String ("                 LargestBlock: ") + String (LargestBlock));
+#endif // def DUMP_FSEQ_HEADER
+            if (0 == TotalChannels)
+            {
+                LastFailedPlayStatusMsg = (String (F ("ParseFseqFile:: Ignoring Range Info. ")) + PlayItemName + F (" No channels defined in Sparse Ranges."));
+                logcon (LastFailedPlayStatusMsg);
+                memset ((void*)&SparseRanges, 0x00, sizeof (SparseRanges));
+                SparseRanges[0].ChannelCount = fsqParsedHeader.channelCount;
+            }
+
+            else if (TotalChannels > fsqParsedHeader.channelCount)
             {
                 LastFailedPlayStatusMsg = (String (F ("ParseFseqFile:: Ignoring Range Info. ")) + PlayItemName + F (" Too many channels defined in Sparse Ranges."));
                 logcon (LastFailedPlayStatusMsg);
                 memset ((void*)&SparseRanges, 0x00, sizeof (SparseRanges));
                 SparseRanges[0].ChannelCount = fsqParsedHeader.channelCount;
             }
 
+            else if (LargestOffset > fsqParsedHeader.channelCount)
```

Comment:
There is no such thing as largest offset as the offset to us is always 0, not calculated

Reply:
Agreed. Fix has been pushed